diesel/src/pg/types/numeric.rs

```diff
@@ -94,7 +94,10 @@ mod bigdecimal {
 
             // Ensure that the decimal will always lie on a digit boundary
             for _ in 0..(4 - scale % 4) {
-                integer = integer * 10;
+                // NOTE: Assignment to `mul` is intended to avoid `#[deny(assign_op_pattern)]` on nightly
+                //       despite lack of `std::ops::MulAssign` impl for `BigInt`.
```

Comment:
Can we just add an `allow` for that instead?

Reply:
Either way, this seems unrelated to your change, and should probably be done in a PR that bumps clippy, not here